pygdf/cudautils.py

```diff
@@ -300,8 +301,9 @@ def gpu_mask_from_devary(ary, bits):
 
 def mask_from_devary(ary):
     bits = make_mask(len(ary))
-    gpu_fill_value.forall(bits.size)(bits, 0)
-    gpu_mask_from_devary.forall(bits.size)(ary, bits)
+    if bits.size > 0:
```

Comment:
is this function ensuring that the bitmasks are 64 byte padded as in the arrow spec? Or if not that at least 64 bit aligned since there are places in the code that assume this so that we can read 64 values at a time from memory?

Reply:
No, currently it seems like it's not padded and uses 1 byte as underlying type. @jrhemstad these are functions we'd want to move to the bit utils anyway, correct?

Reply:
@felipeblazing we're working on updating the `gdf_valid_type` and associated bit utility functions. I believe @BradReesWork can comment on that effort.

Reply:
ok great!